[DAT-2] Create what-we-do-now page

spyrosbek opened the pull request:
### Useful Links

- [JIRA Ticket](https://ails-lab.atlassian.net/browse/DAT-2)
- [Relevant Document](https://docs.google.com/document/d/1WKEMvXAO_yhDg1BMix3BNlzXBF9-SNdvqbk0iSQwNy0/edit#heading=h.dlg3nbm7c2jv)

### How and Why

Create a dedicated page showcasing what the team is currently working on.

### How It Looks Like


https://github.com/datoptron/datoptron-website/assets/24510625/5b26634e-2b4d-4773-8563-351d3fbdde02

spyrosbek committed:
[DAT-2] Create what-we-do-now page

vagkout commented:
Looks pretty nice! Thanks man!

Two notes:
- A typo on the first project (ASK**S**A)
- The blue color for learn-more links does not look so good. Maybe just use black text and underline it?
e.g. similar to what we do here
 
<img width="398" alt="Screenshot 2024-02-07 at 2 03 16 PM" src="https://github.com/datoptron/datoptron-website/assets/11678323/4267d7ac-c284-46af-ae43-405ff53375f6">

spyrosbek committed:
[DAT-2] Styling changes and fix typo

spyrosbek commented:
@vagkout Thanks!
Pushed some changes:
- Fixed ASKSA typo.
- Changed the style of the "Learn more" link (as shown below).
- Changed the style of "now page" link at the header (as shown below).
- Fixed layout responsiveness for mobile devices.

<img width="1349" alt="image" src="https://github.com/datoptron/datoptron-website/assets/24510625/0c7caab7-8407-4023-bab9-63e11e6e1e03">

<img width="494" alt="image" src="https://github.com/datoptron/datoptron-website/assets/24510625/11b01571-a411-40f4-b591-e298b49b2fe0">
